src/zui/ZUIBarDiagram/index.tsx

```diff
@@ -39,11 +38,22 @@ const ZUIBarDiagram = ({ values, size }: ZUIBarDiagramProps) => {
 
   const height = sizes[size];
 
+  const colors = [
+    theme.palette.data[900],
+    theme.palette.data[300],
+    theme.palette.data[100],
+  ];
+  colors.splice(
+    1,
+    0,
+    values.length == 3 ? theme.palette.data[700] : theme.palette.data[500]
+  );
```

Comment:
I don't fully understand this logic and I'm not sure if it's just because I'm tired. Lets discuss tomorrow!

Reply:
I know, it's weird af. I had a moment. 

I'll update to make it more clear, hang on.